### PR TITLE
create Footer and Unit Tests 

### PR DESCRIPTION
**Pull request by yeibercano**
I am adding the Footer component with Unit Tests along with the testing environment.

I am not sure if the Component file-structure is what we want, so please let me know. I can add the the Header Component.

### CODE REVIEW

**Comment by yeibercano** [comment]
screen shot 2017-07-18 at 4 16 59 pm

**Review by richHong — APPROVED** [review]
Change hard-coded values in props. LGTM

**Review by yeibercano — COMMENTED** [review]
@richHong, I refactored it. Still, we may need to refactor this when our file structure is in place. For now, this maybe a good placeholder.

**Comment by yeibercano** [comment]
screen shot 2017-07-19 at 12 11 25 pm

File: package.json [hunk 1]
```diff
@@ -318,7 +318,6 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-preset-node6": "^11.0.0",
-    "babel-preset-stage-2": "^6.24.1",
```
Comment (yeibercano) [inline package.json]: @richHong, yeah. We don't need this. 👍

**Comment by richHong** [comment]
@yeibercano I merged your other PR's but there are conflicts now. Please fix and go ahead merge when ready. 👍

**Comment by yeibercano** [comment]
Great.

Probabbly something with .gitignore file.
Ill check it out when i get home. Thank yous verys muchos

**Pull request merged**